### DISCUSSION TITLE
Zkteco NP1500A Napco attendance machine support

### DISCUSSION THREAD

**Discussion by gfalcon86** [horilla-opensource/horilla#389] [Q&A]
Hello,
we have a "Zkteco NP1500A Napco" attendance and its public device in our country.
when i try to add it even if the login details is right it give me unsuccessful connection so i think its not supported yet.
could you please support it ?

**Comment by horilla-opensource** [comment]
Hi @gfalcon86 ,

Can you please contact us in skype at teamhorilla@gmail.com?

With Regards,
**Team Horilla**

**Reply by gfalcon86** [reply]
Hello @horilla-opensource ,
skype is not working in our country by Gov
could we contact by email messages?
i sent you email message

**Reply by horilla-opensource** [reply]
Hi @gfalcon86 ,

Please contact our biometric developer at sreethul@cybrosys.info. He'll help you out with the same.

**Reply by gfalcon86** [reply]
hi @horilla-opensource 
i will contact him by mail , thank you .
